packages/serialize/README.md

```diff
@@ -0,0 +1,31 @@
+<div align="center">
+
+![Josh Logo](https://evie.codes/josh-light.png)
+
+# @joshdb/serialize
+
+**A serialize utility for the Josh project**
+
+[![GitHub](https://img.shields.io/github/license/josh-development/utilities)](https://github.com/josh-development/utilities/blob/main/LICENSE)
```

Comment:
Could we make this link go directly to /serialize in utilities?

Reply:
This is for the license, which is in the root, not sub-folder